test/collection-facets/facets-template.test.ts

```diff
@@ -66,14 +52,12 @@ describe('Render facets', () => {
     });
   });
 
-  it('emits facet click and change events when a facet is selected/deselected', async () => {
+  it('emits facet click event when a facet is selected/deselected', async () => {
     const facetClickSpy = sinon.spy();
-    const facetChangeSpy = sinon.spy();
     const el = await fixture<FacetsTemplate>(
       html`<facets-template
         .facetGroup=${facetGroup}
         @facetClick=${facetClickSpy}
-        @selectedFacetsChanged=${facetChangeSpy}
```

Comment:
This event no longer exists on the `facet-template` component -- tests updated accordingly.